gratipay/main.py

```diff
@@ -109,10 +109,10 @@
     tell_sentry,
     algorithm['get_response_for_exception'],
 
-    gratipay.set_misc_headers,
+    gratipay.set_version_header,
     authentication.add_auth_to_response,
     csrf.add_token_to_response,
-    cache_static.add_caching_to_response if website.cache_static else noop,
```

Comment:
The reason we're removing the `noop` path here is that we always want to add the proper caching headers to the response, whether or not `cache_static` is turned on. `add_caching_to_response` isn't a noop when we don't have an `etag`, it adds `Cache-Control: no-cache`.